pkg/rook/client/client.go

```diff
@@ -123,11 +138,12 @@ func (a *RookNetworkRestClient) Do(method, query string, body io.Reader) ([]byte
 		return nil, err
 	}
 
-	if response.StatusCode != http.StatusOK {
+	code := response.StatusCode
+	if code != http.StatusOK && code != http.StatusCreated && code != http.StatusNoContent {
```

Comment:
why are these other status codes special, instead of handling them the same way we do for IsHttpAccepted